docs/mkdocs.yml

```diff
@@ -190,13 +191,46 @@ nav:
       - 'Data Collection': 'contributing/data-collection.md'
       - 'Advocating': 'contributing/advocating.md'
       - 'Maintainers': 'contributing/maintainers.md'
-  - 'References':
-      - 'Static Configuration':
-          - 'Overview': 'reference/static-configuration/overview.md'
-          - 'File': 'reference/static-configuration/file.md'
-          - 'CLI': 'reference/static-configuration/cli.md'
-          - 'Environment variables': 'reference/static-configuration/env.md'
-      - 'Dynamic Configuration':
+  - 'Reference':
+    - 'Install Configuration':
+      - 'Boot Environment': 'reference/install-configuration/boot-environment/overview.md'
+      - 'Configuration Discovery':
+        - 'Overview' : 'reference/install-configuration/providers/overview.md'
+        - 'Kubernetes':
+          - 'Kubernetes Gateway API' : 'reference/install-configuration/providers/kubernetes/kubernetes-gateway.md'
+          - 'Kubernetes CRD' : 'reference/install-configuration/providers/kubernetes/kubernetes-crd.md'
+          - 'Kubernetes Ingress' : 'reference/install-configuration/providers/kubernetes/kubernetes-ingress.md'
+        - 'Docker': 'reference/install-configuration/providers/docker.md'
+        - 'Swarm': 'reference/install-configuration/providers/swarm.md'
+        - 'Hashicorp':
+          - 'Nomad': "reference/install-configuration/providers/hashicorp/nomad.md"
+          - 'Consul': 'reference/install-configuration/providers/hashicorp/consul.md'
+          - 'Consul Catalog': 'reference/install-configuration/providers/hashicorp/consul-catalog.md'
+        - 'KV Stores': 
+          - 'Redis': 'reference/install-configuration/providers/kv/redis.md'
+          - 'Consul': 'reference/install-configuration/providers/kv/consul.md'
+          - 'etcd': 'reference/install-configuration/providers/kv/etcd.md'
+          - 'ZK' : 'reference/install-configuration/providers/kv/zk.md'
+        - 'Others':
```

Comment:
I find "others" a bit negative, what was the rationale for creating this section @sheddy-traefik @nmengin ?

Reply:
Why not putting those at the upper level as Docker and Swarm?

Reply:
@rtribotte "others" was what we concluded with Emile in the new structure for the documentation. However, while i get your concern, i think it's fine in my own opinion because they are the providers that you can't fit into a specific category. I'll let @nmengin share his thoughts too

Reply:
@sheddy-traefik I understand they don't fit in any category, but my question is more why they are not being as "promoted" as Docker and Swarm?

Reply:
Ah i see, @nmengin would be the best person to answer this as most of the new structure of the doc was concluded before I came in.